script.js

```diff
@@ -0,0 +1,51 @@
+/*
+- `next()`: returns the highest priority item from the list in the same format as `getItem`
+(if there are several items of the same priority, return the one that was added before others; if there are no items in the list, throw an error).
+ */
+ class TODO {
+    constructor() {
+        this.list = [];
+        this.idCounter = 1;
+        console.log(`Created new TODO list`);
+    }
+     addItem(title, priority) {
+        const newItem = {title: title, priority: priority, id: this.idCounter};
+        this.list.push({title: title, priority: priority, id: this.idCounter});

```

Comment:
такий запис буде кращим
```suggestion
        this.list.push(newItem);
```